--- settings.py	2014-03-26 17:02:32.754121608 +0000
+++ settings_production.py	2014-03-26 17:02:04.910121871 +0000
@@ -21,11 +21,11 @@
 SECRET_KEY = '_-j^f1$)wi7sln)zsh)#yse0+nv=v5e^%9v!$eh=a6cus69x96'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
 
-TEMPLATE_DEBUG = True
+TEMPLATE_DEBUG = False
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['kanga-mhha1g11.ecs.soton.ac.uk']
 
 
 # Application definition
